src/pages/instant-observability.js

```diff
@@ -23,6 +23,8 @@ import CATEGORIES from '../data/instant-observability-categories';
 
 import SuperTiles from '../components/SuperTiles';
 
+const COLLAPSE_BREAKPOINT = '760px';
```

Comment:
Since we're using this in multiple components now, we should break it out into the constants file. Maybe rename it to `QUICKSTARTS_COLLAPSE_BREAKPOINT`

Reply:
Oooo i like it. I'll get to it.